src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/transformer/cas2/ApplicationsTransformerTest.kt

```diff
@@ -85,35 +90,64 @@ class ApplicationsTransformerTest {
     () {
       val application = object : Cas2ApplicationSummary {
         override fun getId() = UUID.fromString("2f838a8c-dffc-48a3-9536-f0e95985e809")
-        override fun getCrn() = randomStringMultiCaseWithNumbers(6)
+        override fun getCrn() = "CRN456"
         override fun getCreatedByUserId() = UUID.fromString("836a9460-b177-433a-a0d9-262509092c9f")
         override fun getCreatedAt() = Timestamp(Instant.parse("2023-04-19T13:25:00+01:00").toEpochMilli())
         override fun getSubmittedAt() = null
+        override fun getNomsNumber() = "NOMS123"
+        override fun getPncNumber() = "PNC321"
+        override fun getName() = "Robert Smith"
+        override fun getDateOfBirth() = LocalDate.now().minusYears(60)
+        override fun getNationality() = "British"
+        override fun getSex() = "Male"
+        override fun getPrisonName() = "HMP Bristol"
+        override fun getPersonStatus() = Cas2NewApplication.PersonStatus.inCustody.name
       }
 
+      val person = FullPerson(
```

Comment:
We could have a test that tests that a person with none of the optional fields handles correctly, presumably returning a `FullPerson` with null for some attributes, which sounds a bit confusing because you might expect an object called 'full' to have everything.